codalab/bundles/run_bundle.py

```diff
@@ -31,6 +31,8 @@ class RunBundle(DerivedBundle):
     METADATA_SPECS.append(MetadataSpec('request_memory', str, 'Amount of memory (e.g., 3, 3k, 3m, 3g, 3t) allowed for this run.', formatting='size', default='2g'))
     METADATA_SPECS.append(MetadataSpec('request_disk', str, 'Amount of disk space (e.g., 3, 3k, 3m, 3g, 3t) allowed for this run. Defaults to user disk quota left.', formatting='size', default=None))
     METADATA_SPECS.append(MetadataSpec('request_cpus', int, 'Number of CPUs allowed for this run.', default=1))
+    METADATA_SPECS.append(MetadataSpec('cpu_usage', str, 'Portion of CPU used by this container (e.g., 0.28)', generated=True))
```

Comment:
Should this be a percentage instead?

Reply:
I am ok with either way. If you think percentage represents better, I can definitely do it.